### ISSUE TITLE
Cannot read property 'pointSize' of null

### ISSUE THREAD

**Issue by vallemar** [nativescript-community/ui-label#9]
### Which platform(s) does your issue occur on?
- iOS
- iOS: 14.3
- emulator

### Please, provide the following version numbers that your issue occurs with:

- CLI: 8.0.1
- @nativescript-community/ui-label": "^1.1.10
- @nativescript/ios": "8.0.0

### Please, tell us how to recreate the issue in as much detail as possible. 
When I try to use HTMLLabel it gives me the following exception:

```
***** Fatal JavaScript exception - application has been terminated. *****
NativeScript encountered a fatal error: Uncaught TypeError: Cannot read property 'pointSize' of null
at
updateHTMLString(file: src/webpack:/checkpointcurly/node_modules/@nativescript-community/ui-label/label.ios.js:349:0)
at _setNativeText(file: src/webpack:/checkpointcurly/node_modules/@nativescript-community/ui-label/label.ios.js:443:0)
at onResumeNativeUpdates(file: src/webpack:/checkpointcurly/node_modules/@nativescript-community/ui-label/label-common.js:44:0)
at _resumeNativeUpdates(file: src/webpack:/checkpointcurly/node_modules/@nativescript/core/ui/core/view-base/index.js:280:0)
at onLoaded(file: src/webpack:/checkpointcurly/node_modules/@nativescript/core/ui/core/view-base/index.js:235:0)
at onLoaded(file: src/webpack:/checkpointcurly/node_modules/@nativescript-community/ui-label/label.ios.js:159:0)
at (file: src/webpack:/checkpointcurly/node_modules/@nativescript/core/ui/core/view-base/index.js:318:0)
at callFunctionWithSuper(file: src/webpack:/checkpointcurly/node_modules/@nativescript/core/ui/core/view-base/index.js:312:0)
at callLoaded(file: src/webpack:/checkpointcurly/node_modules/@nativescript/core/ui/core/view-base/index.js:318:0)
at loadView(file: src/webpack:/checkpointcurly/node_modules/@nativescript/core/ui/core/view-base/index.js:458:0)
at (file: src/webpack:/checkpointcurly/node_modules/@nativescript/core/ui/core/view-base/index.js:237:0)
at eachChildView(file: src/webpack:/checkpointcurly/node_modules/@nativescript/core/ui/layouts/layout-base-common.js:101:0)
at eachChild(file: src/webpack:/checkpointcurly/node_modules/@nativescript/core/ui/core/view/view-common.js:759:0)
at onLoaded(file: src/webpack:/checkpointcurly/node_modules/@nativescript/core/ui/core/view-base/index.js:236:0)
at (file: src/webpack:/checkpointcurly/node_modules/@nativescript/core/ui/core/view-base/index.js:318:0)
at callFunctionWithSuper(file: src/webpack:/checkpointcurly/node_modules/@nativescript/core/ui/core/view-base/index.js:312:0)
at callLoaded(file: src/webpack:/checkpointcurly/node_modules/@nativescript/core/ui/core/view-base/index.js:318:0)
at loadView(file: src/webpack:/checkpointcurly/node_modules/@nativescript/core/ui/core/view-base/index.js:458:0)
at (file: src/webpack:/checkpointcurly/node_modules/@nativescript/core/ui/core/view-base/index.js:237:0)
at eachChildView(fi<…>
```

It's only happening to me on iOS, on android it works fine. I am also in NS 8 (I had not tried this plugin before so I could not tell if it only happens in 8). My code:

```
  <HTMLLabel
          width="100%"
          height="50"
          :autoFontSize="true"
          :textWrap="true"
          :html="username"
          maxFontSize="30"
        />
```

**Comment by farfromrefug** [comment]
@vallemar should be fixed in 1.1.11

**Comment by vallemar** [comment]
@farfromrefug Now the error has been solved but the line skipping does not do well. When the text overflows it jumps to the next line.

**Comment by farfromrefug** [comment]
it worked here. can you show?

**Comment by vallemar** [comment]
Hi @farfromrefug, I have tried the following two settings: textWrap = true and textWrap = false. The result is the same. I show an image of how it looks in both cases: (the text is Lorem Ipsum is....)

![image](https://user-images.githubusercontent.com/15719383/113460031-9df26800-9417-11eb-89ac-00fd5ca30360.png)


The code:
```
  <HTMLLabel
                width="100%"
                :autoFontSize="true"
                :textWrap="false"
                :html="username"
                maxFontSize="30"
              />
```

**Comment by farfromrefug** [comment]
@vallemar i honestly dont see what s wrong here. What are you expecting?

**Comment by vallemar** [comment]
Ah OK, the problem is that it does not respect the TextWrap property, the text should resize in 1 line and as seen in the previous image there are 3 lines

**Comment by farfromrefug** [comment]
what if you use "text" instead of "html" ? don't use "html" if you don't have html code

**Comment by vallemar** [comment]
ah ok, sorry I hadn't seen the text property. Anyway, the behavior is the same as with html, the textWrap property does not respect it, even with text I don't see that the text is resized

**Comment by farfromrefug** [comment]
@vallemar should be fixed in 1.1.12
